src/arb/acb.jl

```diff
@@ -843,7 +843,7 @@ end
 @doc Markdown.doc"""
     expm1(x::acb)
 
-Return $\exp(x)-1$, evaluated accurately for small $x$.
+Return $\exp(x)-1$, using an accurate method when $x \approx 0$.
```

Comment:
Or perhaps "a more accurate method"

Reply:
Yeah, I like "a more accurate method". 

I don't see a major problem with the original wording, but if this is clearer to you then let's put this in.